setup.py

```diff
@@ -8,10 +8,19 @@
 
 def get_requirements():
     """Parse all packages mentioned in the 'requirements.txt' file."""
-    with open('requirements.txt') as fd:
-        return fd.read().splitlines()
+    with open('requirements.in') as fd:
+        lines = fd.read().splitlines()
+        reqs, dep_links = [], []
+        for line in lines:
+            if line.startswith('git+'):
+                dep_links.append(line)
+            else:
+                reqs.append(line)
+        return reqs, dep_links
```

Comment:
This is incorrect, can you refer https://github.com/fabric8-analytics/fabric8-analytics-utils/pull/58? dependency_links is deprecated.